docs/api-configuration.rst

```diff
@@ -15,8 +15,11 @@ published protocol buffers.
 This plugin *will* successfully publish a library on a valid protobuf API
 even without any additional information set, but may require some
 post-processing work by a human in this case before the resulting client
-library will install or work. (Look for values enclosed by ``<<<`` and
-``>>>`` to quickly spot these.)
+library will work.
+
+Look for values enclosed by ``<<<`` and ``>>>`` to quickly spot these.
+As of this writing, these are the ``SERVICE_ADDRESS`` and ``OAUTH_SCOPES``
```

Comment:
I think `SERVICE_ADDRESS` is called `HOSTNAME` now. Also check oauth scopes name.

Reply:
Nope, it is still `SERVICE_ADDRESS` in the client library.